src/middlewares/auth/utils/isAuthHandlerFromParams.js

```
@@ -0,0 +1,16 @@
+const UnauthorizedError = require("../../../exceptions/UnauthorizedError");
+const { Member } = require("../../../models");
+
+const isAuthHandlerFromParams = async (req, res) => {
+  const memberId = req.params.memberId;
+  //console.log("memberId from body:", memberId);
+  const member = await Member.findById(memberId);
+  if (!member) return false;
```

IsAuthHandler should throw Error (instead of returning false).